Add auth notice to installation instructions (#87)
codidact/core: docs/linux-instructions.md

 3. You can launch the application by executing `dotnet run` in the `src/WebUI` directory.

 4. You can run the tests by executing `dotnet test` in the root directory.
 

>  ## Setting up the auth provider

Reviewer: ```suggestion
 ## Setting Up The Authentication Provider
```

Author: I'll not change "up the" for consistency with the other headings.

>  You'll also need to set up the auth provider. See [this instructions](https://github.com/codidact/authentication/blob/develop/docs/development.md) for more information.

Reviewer: I am not a native speaker, is "those instructions" correct?
```suggestion
You will also need to set up the authentication provider. See [those instructions](https://github.com/codidact/authentication/blob/develop/docs/development.md) for more information.
```

Reviewer 2: Isnt it these instructions?

Reviewer: I just read an article about those *versus* these and I still do not know.

Author: I am not a native speaker either, but "these" sounds more correct IMO. I'm going to make this change and then merge.
